src/pyhf/readxml.py

```diff
@@ -23,13 +23,14 @@ def extract_error(h):
     bin uncertainties are then Poisson, and so the `sqrt(entries)`.
 
     Args:
-        h (uproot.rootio.TH1 object): The histogram
+        hist (:class:`uproot.behaviors.TH1.TH1`): The histogram
```

Comment:
@jpivarski can you confirm is this is what you'd expect users to understand (documentation-wise)?

Reply:
Yes, that's the most useful way to annotate the type of the histogram. It's the superclass that is persistent, documented, and has all of the user-oriented methods defined.

It should point here: https://uproot.readthedocs.io/en/latest/uproot.behaviors.TH1.TH1.html (if the intersphinx stuff works as expected).

Reply:
renders as <img width="700" alt="Screen Shot 2021-01-05 at 10 33 01 AM" src="https://user-images.githubusercontent.com/761483/103668386-06c76280-4f45-11eb-97eb-7fc9fc44859e.png">

and links to https://uproot.readthedocs.io/en/latest/uproot.behaviors.TH1.TH1.html#uproot.behaviors.TH1.TH1